pkgs/tools/misc/wev/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl
+, pkg-config, scdoc
+, wayland, wayland-protocols, libxkbcommon
+}:
+
+let
+  version = "2019-08-11";
+  commit = "47d17393473be152cf601272faf5704fff1c3f92";
+in stdenv.mkDerivation {
+  pname = "wev-unstable";
```

Comment:
Please mark the ``pname`` as ``wev`` and include the ``unstable`` in the version please.

Reply:
Does this have any advantage?

IMO this variant makes more sense as the version in NixOS always begins with a digit and it would also be consistent with `parseDrvName`:
```
nix-repl> builtins.parseDrvName "wev-unstable-2019-08-11"
{ name = "wev-unstable"; version = "2019-08-11"; }
```

From the Nixpkgs manual:
> - The version part of the name attribute must start with a digit (following a dash) — e.g., "hello-0.3.1rc2".
> - If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

but it would still "violate" the following (assuming it actually refers to the `pname` part):
> - The name attribute should be identical to the upstream package name.

Though I see that having `-unstable` in `pname` looks a bit strange/wrong.

Reply:
I have always assumed ``pname`` has been introducing for exactly this reason, to have a unique and stable package name. Sorry for confusion.

Reply:
@mmahut yeah, I guess it's controversial atm (there are arguments for and against each variant) and we don't have an official rule for it yet (afaik).

I did look into it a bit more and it seems like most packages actually do put it in the `version` attribute atm:
```
michael@jarvis> git rev-parse HEAD
79c6d22c1fc55b69961f5b72c3b293fc739d0ef6
michael@jarvis> git grep -E '(pname|version) = ".*unstable' | grep -oE "pname|version" | sort | uniq -c
     10 pname
    156 version
```

But some of the dedicated PRs ("treewide: name -> pname") also put it into `pname`.

With a quick search I found the following discussion regarding this: https://github.com/NixOS/nixpkgs/issues/61691#issuecomment-493774141

Reply:
@primeos yes, that's why I have been confused about it. But we have a rule - it's in the manual like you used it :)